app/models/setting.rb

```diff
@@ -268,4 +270,10 @@ def refresh_registry_value
       definition.value_from_db = value
     end
   end
+
+  def remove_whitespaces
+    self[:value] = value.each { |a| a.strip! if a.respond_to? :strip! }
```

Comment:
this gets executed for *any* kind of setting, but I think it needs to only happen for settings of type "array", as otherwise `.each` is not defined for those